CoreImportExportPlugins/nbproject/project.xml

```diff
@@ -14,6 +14,14 @@
                         <specification-version>1.0</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>au.gov.asd.tac.constellation.functionality</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <specification-version>1.0.7</specification-version>
```

Comment:
if this dependency was added as a result of the welcome page then it can probably be removed